server/modules/allocations.js

```diff
@@ -21,7 +24,8 @@ export const placementStatus = {
 
 // Updates the local redis store with latest Partner List
 export const updatePartnerStore = async () => {
-  axios.get('api/v1/partners')
+  axios
```

Comment:
since this is an `async` function, it'll make sense to use `async await` as oppose to promises.

Reply:
This change is not part of my implementation. Was highlighted due to automatic code formatting